### DISCUSSION TITLE
Alternatives to Spectre and Nordigen for US-based users? 🦅🤕

### DISCUSSION THREAD

**Discussion by cwmoriarty** [firefly-iii/firefly-iii#7765] [Firefly III]
I stood up Firefly-iii selfhosted (docker arm64) and am having some issues with getting the tool up and functioning for my purposes. I won't be able to make use of Firefly-iii unless I can make progress on the below two:

### Historical transactions
I've exported a transactions.csv from Mint but, even with a tiny subselection deduped, the data-importer tool always seems to choke up, even if I onlu select 1-2 columns. Date format is correct. It's authenticated to the other container fine. Nothing obvious in the logs, and there are only 4-5 rows in my CSV. I have it pushing to a single bank that I configured with the Firefly-iii wizard.

It would be helpful if someone had specific guidance on how to cleanly get Mint historicals into Firefly-iii!

### New transactions
After registering and investigating Spectre (Saltedge) and Nordigen (Gocardless::Bank Account Data) there are no American banks available (seemingly by design). I understand that American banks are all evil to varying degrees. Are there any recommendations for Americans? Plaid integration? 

Thanks!

**Comment by JC5** [comment]
There's a plaid importer for Firefly III yes, but otherwise there's not much, I know :(.

Salt Edge used to support American banks but their support stopped a few years ago. I've made a reminder to put this in the docs.

**Reply by cwmoriarty** [reply]
Note for the thread: https://github.com/GeorgeHahn/firefly-plaid-connector

**Reply by cskujawa** [reply]
They've released a new version as well mentioned in another discussion:
https://github.com/orgs/firefly-iii/discussions/7065#discussion-4884099
